### DISCUSSION TITLE
Bemühungen um erneuerbare Energien auch visualisieren

### DISCUSSION THREAD

**Discussion by silberzwiebel** [codeformuenster/klimawatch#123] [Ideas]
Oft Teil von Klimaschutzkonzepten: CO2-Reduktion *und* Ausbauziele erneuerbare Energien.

**Comment by silberzwiebel** [comment]
@kolossos: Es könnte sein, dass Du Ahnung hierzu hast: Wie könnten wir am besten den Ausbaustand erneuerbarer Energien visualisieren?

**Comment by kolossos** [comment]
Mir fällt da spontan die Grfik von Prof. Quaschning ein:
https://www.volker-quaschning.de/grafiken/2015-12_Dekarbonisierung-D/index.php
Vielleicht auch diese:
https://www.volker-quaschning.de/grafiken/2013-12_Bruttostrom_Zielkorridor/index.php
Daraus leitet der Professor dann bestimmte Zubauraten für Erneuerbare ab, Photovoltaik zum Teil mit >11GW/a. Das sind jetzt aber alles noch keine tollen bzw. interaktiven Visualisierungen. Irgendwo sah ich im Web mal eine schöne Simulation mit vielen Schiebereglern worüber man den zukünftigen CO2-Ausstoss prognostizieren konnte, find die Seite aber nicht mehr.

**Comment by silberzwiebel** [comment]
Linksammlung, damit es nicht verlorengeht, mit Datenquellen/Visualisierungsideen:
- https://www.energieatlas.nrw.de/site/wms-und-wfs-dienste (API-Link, dort auch schon einige Visualisierungen)
- http://www.energymap.info/

**Comment by jonek** [comment]
Direkter Link zur "Bestandskarte Strom" aus NRW: https://www.energieatlas.nrw.de/site/bestandskarte

Die Visualisierung der PV-Anlagen scheint die Daten aus dem MaStR (Markstammdatenregister) zu benutzen.
In MaStR sind grosse PV-Anlagen mit Geo-Koordinaten und kleine Anlagen mit Postleitzahl erfasst.
Die grossen Anlagen sind auf der Karte anscheinend unter ihren Lat, Lon Koordinaten dargestellt.
Die kleinen Anlagen werden in einer Liste im Mittelpunkt(?) der Postleitzahlenregion zusammengefasst.

**Comment by jonek** [comment]
Für Baden-Württemberg gibt es ebenfalls eine interaktive PV-Karte. Allerdings nur mit einem Datenstand von 2013.
https://www.energieatlas-bw.de/sonne/dachflachen/bestehende-dachflachenanlagen

Aktueller Daten zur Photovoltaik in Baden-Württemberg sind hier zu finden: https://www.photovoltaik-bw.de/pv-netzwerk/pv-themen/photovoltaik-liga-baden-wuerttemberg/ Allerdings sind diese nicht maschinenlesbar oder als Open Data bereitgestellt. Auch sind die Karten dort nicht interaktiv. Als Quellen werden die Bundesnetzagentur (wahrscheinlich das Markstammdatenregister) und die LUBW (Landesanstalt für Umwelt Baden-Württemberg) angegeben.
